src/OpenRiaServices.Client/Framework/Internal/MergeAttribute.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 
-namespace OpenRiaServices.Server.Data
+namespace OpenRiaServices.Client.Internal
 {
     [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, AllowMultiple = false, Inherited = true)]
     public class MergeAttribute : Attribute
```

Comment:
```suggestion
    class MergeAttribute : Attribute
```

Reply:
I am considering removing the attribute since IgnoreDataMember can be used instead.
It might however be better to do so in a separate PR so that commit can easily be reverted if we must add it back